jvm-packages/xgboost4j/src/main/scala/ml/dmlc/xgboost4j/scala/rabit/RabitTracker.scala

```diff
@@ -93,8 +93,11 @@ private[scala] class RabitTracker(numWorkers: Int, port: Option[Int] = None,
     * @return Boolean flag indicating if the Rabit tracker starts successfully.
     */
   private def start(timeout: Duration): Boolean = {
+    val hostAddress = Option(TrackerProperties.getInstance().getHostIp)
+      .map(InetAddress.getByName).getOrElse(InetAddress.getLocalHost)
```

Comment:
we do not include property file in the released jar, you mean build by your own?

Reply:
`TrackerProperties` already handles the case if the property file is missing in the jar. In that case it would return `null` when calling `getHostIp` hence the `Option`. 

I borrowed this idea from the java RabitTracker implementation [here](https://github.com/dmlc/xgboost/blob/master/jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/RabitTracker.java#L147).

Reply:
I know, I mean how you add a property file , you have to build by your own

Reply:
I didn't try adding in application's resource folder

Reply:
Correct. In my project I have to add `resources/xgboost-tracker.properties` file for example with the following contents:
```
host-ip=0.0.0.0
```

Reply:
Good!